readme.md

```diff
@@ -113,19 +113,236 @@ can be found in our Kubernetes configuration https://github.com/zalando-incubato
 - Extending It with Customized Predicates, Filters, and Builds
-- Proxy Packages
-- Logging and Metrics
+- [Predicates](https://godoc.org/github.com/zalando/skipper/predicates) - additional predicates to match a route
+- [Proxy Packages](https://godoc.org/github.com/zalando/skipper/proxy)
+- [Logging](https://godoc.org/github.com/zalando/skipper/logging) and [Metrics](https://godoc.org/github.com/zalando/skipper/metrics)
 - Performance Considerations
-- Rate Limiters
-
+- [Rate Limiters](https://godoc.org/github.com/zalando/skipper/filters/ratelimit)
+
+#### 1 Minute Skipper introduction
+
+The following example shows a skipper routes file in eskip format, that has 3 named routes: baidu, google and yandex.
+
+    % cat doc-1min-intro.eskip
+    baidu:
+            Path("/baidu")
+            -> setRequestHeader("Host", "www.baidu.com")
+            -> setPath("/s")
+            -> setQuery("wd", "godoc skipper")
+            -> "http://www.baidu.com";
+    google:
+            *
+            -> setPath("/search")
+            -> setQuery("q", "godoc skipper")
+            -> "https://www.google.com";
+    yandex:
+            * && Cookie("yandex", "true")
+            -> setPath("/search/")
+            -> setQuery("text", "godoc skipper")
+            -> tee("http://127.0.0.1:12345/")
+            -> "https://yandex.ru";
+
+Matching the route:
+
+- baidu is using Path() matching to differentiate the HTTP requests to select the route.
+- google is the default matching with wildcard '*'
+- yandex is the default matching with wildcard '*' if you have a cookie "yandex=true"
+
+Request Filters:
+
+- If baidu is selected, skipper sets the Host header, changes the path and sets a query string to the http request to the backend "http://www.baidu.com".
+- If google is selected, skipper changes the path and sets a query string to the http request to the backend "https://www.google.com".
+- If yandex is selected, skipper changes the path and sets a query string to the http request to the backend "https://yandex.ru". The modified request will be copied to "http://127.0.0.1:12345/"
+
+Run skipper with the routes file doc-1min-intro.eskip shown above
+
+    % skipper -routes-file doc-1min-intro.eskip
+
+To test each route you can use curl:
+
+    % curl -v localhost:9090/baidu
+    % curl -v localhost:9090/
+    % curl -v --cookie "yandex=true" localhost:9090/
+
+To see the request that is made by the tee() filter you can use nc:
+
+    [terminal1]% nc -l 12345
+    [terminal2]% curl -v --cookie "yandex=true" localhost:9090/
+
+#### 3 Minutes Skipper in Kubernetes introduction
+
+You should have a base understanding of [Kubernetes](https://kubernetes.io) and
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+
+Prerequisites: First you have to install skipper-ingress as for
+example daemonset, create a deployment and a service.
+
+We start to deploy skipper-ingress as daemonset, use hostNetwork and
```

Comment:
We start to deploy skipper-ingress as _a_ daemonset, use hostNetwork and
expose _the port 9999_ on each Kubernetes worker node for incoming ingress
traffic.